packages/@aws-cdk/aws-servicecatalog/lib/portfolio.ts

```diff
@@ -20,10 +23,11 @@ export interface PortfolioShareOptions {
   readonly shareTagOptions?: boolean;
 
   /**
-   * The accept language of the share
+   * The accept language of the share.
+   * Controls status and error message language for share.
    * @default - accept language not specified
    */
-  readonly acceptLanguage?: AcceptLanguage;
+  readonly acceptedMessageLanguage?: AcceptLanguage;
```

Comment:
How about just calling it `messageLanguage`? I don't get the "accept" part of this name.

Reply:
okay. Will move to `messageLanguage` and add this naming convention as the breaking change